### PR TITLE
Put a start server script that includes a migration

### PR DESCRIPTION
**Pull request by map34**
## Overview

Put a start server script that includes a migration for easy startup.

### CODE REVIEW

**Comment by scaraclette** [comment]
Sipp

File: infra/scripts/start-server.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+source activate TEST;
+./infra/scripts/wait-for-postgres.sh python backend/server/manage.py migrate;
```
Comment (lsetiawan) [inline infra/scripts/start-server.sh:6]: Umm... don't you need a `;` after `./infra/scripts/wait-for-postgres.sh`?

Comment (map34) [reply infra/scripts/start-server.sh]: You don't have to, but yes it bothers me that that thing has no semicolon lmao

Comment (map34) [reply infra/scripts/start-server.sh:6]: No you don't haha, look inside wait-for-postgres script

**Pull request merged**